frame/staking/src/voter_bags/thresholds.rs

```diff
@@ -0,0 +1,229 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Generated voter bag thresholds.
+
+use super::N_BAGS;
+
+/// Ratio between adjacent bags;
+#[cfg(any(test, feature = "std"))]
+#[allow(unused)]
+pub const CONSTANT_RATIO: f64 = 1.2483305489016119;
```

Comment:
This must not be used in any consensus code, right? maybe it can be just `test`?

Reply:
Of course, and it already is `#[cfg(any(test, feature = "std"))]`. Even so, I think it's useful to have the actual generated value present in the codebase so it's easier for humans to understand.